DisguiseUnityRenderStream/Runtime/DisguiseFramerateManager.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+
+namespace Disguise.RenderStream
+{
+    public static class DisguiseFramerateManager
+    {
+#if ENABLE_CLUSTER_DISPLAY
+        public const bool Enabled = false;
+#else
+        public const bool Enabled = true;
+#endif
+        
+        static int k_FrameRateUnlimited = -1;
+        
+        public static bool FrameRateIsLimited => Application.targetFrameRate >= 0;
+
+        public static bool VSyncIsEnabled => QualitySettings.vSyncCount > 0;
+
+        public static void Initialize()
+        {
+#if !ENABLE_CLUSTER_DISPLAY
+            RemoveFrameLimit();
+#endif
+        }
+
+        public static void Update()
+        {
+#if !ENABLE_CLUSTER_DISPLAY
+            if (VSyncIsEnabled)
+            {
+                Debug.LogWarning($"{nameof(DisguiseFramerateManager)}: {nameof(QualitySettings)}{nameof(QualitySettings.vSyncCount)} is enabled and may affect performance.");
```

Comment:
Are you missing a dot in `{nameof(QualitySettings)}{nameof(QualitySettings.vSyncCount)}`? Shouldn't it be
```
{nameof(QualitySettings)}.{nameof(QualitySettings.vSyncCount)}
```

Reply:
Fixed